src/api/index.ts

```diff
@@ -95,6 +93,19 @@ export class Api {
     };
   }
 
+  async generateNftOrderToBeSigned(orderInput: NftOrderInput) {
+    const response = await this.api.generateNftOrderToBeSigned({ orderInput });
+    return response.typedData as TypedDataDefinition;
```

Comment:
Will this help with MM?

Reply:
The as just makes an error go away. Since we sign typed data, it will show on mm